docs/docs/guides/remote-machine-learning.md

```diff
@@ -16,6 +16,8 @@ version: '3.8'
 services:
   immich-machine-learning:
     container_name: immich_machine_learning
+    # For hardware acceleration, add one of -[armnn, cuda, openvino] to the image tag.
+    # Example tag: ${IMMICH_VERSION:-release}-cuda
     image: ghcr.io/immich-app/immich-machine-learning:${IMMICH_VERSION:-release}
     volumes:
```

Comment:
```suggestion
   extends:
      file: hwaccel.ml.yml
      service: # set to one of [armnn, cuda, openvino, openvino-wsl] for accelerated inference - use the `-wsl` version for WSL2 where applicable
    volumes:
```

Reply:
I am not an expert at these but for it to work i did not need to add this line :) someone else can probably give their opinion

Reply:
It will run without that line, but won't actually use CUDA.

Reply:
Is the indentation in that suggestion right? It looks a bit off.

Reply:
> Is the indentation in that suggestion right? It looks a bit off.

I copied it from my computer and it looks fine, maybe it just looks like that on GitHub.

Reply:
I think you need to add another space before `extends`